benchmarks/stemming_sample.txt

```diff
@@ -3251,7 +3251,7 @@
 ΓΕΝΟΚΤΟΝΙΑΣ,ΓΕΝΟΚΤΟΝ
 ΓΕΝΟΣ,ΓΕΝ
 ΓΕΝΟΥ,ΓΕΝ
-ΓΕΝΟΥΑ,ΓΕΝΟΥ
+ΓΕΝΟΥΑ,ΓΕΝΟ
```

Comment:
@m-Peter I think we should treat "ΟΥ" as a special case (diphthong) and not split it. "ΓΕΝΟΥΑ" should probably stem to "ΓΕΝΟΥ".